doc/README.AIX

```diff
@@ -0,0 +1,64 @@
+This file describes how to configure pam_hbac for access control on a
+AIX machine.
+
+Only tested on AIX 7.1 TL4.
+
+Prerequisities
+==============
+Please make sure your AIX client is able to resolve and authenticate
+the IPA or AD users. For example, for users coming from an AD trust:
+    $ id administrator@win.trust.test
+    $ su - administrator@win.trust.test
+A good starting point for this configuration is to read:
+    https://www.freeipa.org/page/ConfiguringUnixClients
+
+Building from source
+====================
+The build environment used to build the module was obtained from the AIX Toolbox
+for Linux Applications (https://www-03.ibm.com/systems/power/software/aix/linux/toolbox/alpha.html).
+It is recommended to use the yum.sh script provided at the top of their webpage,
+it will install the RPM package manager and provide the yum utility 
+(https://ftp.software.ibm.com/aix/freeSoftware/aixtoolbox/ezinstall/ppc/yum.sh) that
+will ensure all dependencies are installed.
+Please make sure all required dependencies are installed. On AIX 7.1 TL4, this
+would be:
+    autoconf-2.69-1 automake-1.15-1 gcc-6.3.0-1 gcc-cpp-6.3.0-1 gettext-0.19.7-1 \
+    glib2-2.48.0-1 libtool-2.4.6-2 m4-1.4.13-1 openldap-2.4.40-2 pkg-config-0.19-6
+
+This build does not have manpages. In AIX the utility a2x does something other than
+converting asciidoc. It must be removed from the PATH variable otherwise the build
+will fail. 
```

Comment:
Hmm, this is unfortunate, do you think it would be a good idea to provide a configure-time switch `--without-manpages`` so that admins on AIX could just skip building the man pages altogether?

Reply:
Yes, totally agree.